hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -721,8 +727,10 @@ private class Emit(
               srvb.offset
             ))))
 
-      case _: ArrayMap | _: ArrayZip | _: ArrayFilter | _: ArrayRange | _: ArrayFlatMap | _: ArrayScan | _: ArrayLeftJoinDistinct | _: RunAggScan | _: ArrayAggScan | _: ReadPartition =>
+      case _: ArrayMap | _: ArrayZip | _: ArrayFilter | _: ArrayRange | _: ArrayFlatMap | _: ArrayScan | _: ArrayLeftJoinDistinct | _: RunAggScan | _: ArrayAggScan | _: ReadPartition => {
```

Comment:
Need to add MakeStream and StreamRange here.

Actually, it seems even better to do `case x if x.typ.isInstanceOf[TStream]`!

Reply:
Was looking at this too. The x.typ check doesn’t work however. In any case still dealing with some failures. Will work on this again in a few hours.